### DISCUSSION TITLE
how to implement 'warn_before_in_seconds' correctly?

### DISCUSSION THREAD

**Discussion by arismetyogi** [niladam/filament-auto-logout#6] [Q&A] [answered]
I set the duration 15mins for auto-logout  and the 'warn_before_in_seconds' of 30 seconds.
but the notification always showing from the full duration, not using the 'warn_before_in_sevonds' value.

**Comment by niladam** [comment, marked as answer]
hello @arismetyogi -- the `warn_before_in_seconds` refers to the time BEFORE sending out the notification.

Essentially, if you set this to 60 seconds, it will send a notification that the logout will happen 1 minute *BEFORE* logging out.

I hope i understood what you're saying ?

**Reply by arismetyogi** [reply]
I see, got it, thanks so much.
Originally, I thought that the timer on the top bar is the notification, actually I found out that the notification is a different thing.

That solved my question.

**Reply by niladam** [reply]
You're welcome!:)
